s3-shield/cloud-init/cloud-init-s3-shield.yaml.tmpl

```diff
@@ -15,7 +18,7 @@ write_files:
     Environment="AWS_CONFIG_FILE=/etc/varnish/s3.conf"
     ExecStart=
     ExecStart=/usr/sbin/varnishd -a :80 -a 127.0.0.1:8443,proxy -S /etc/varnish/secret -T localhost:6082 -f /etc/varnish/s3shield.vcl -s mse -p syslog_cli_traffic=off
-  path: /etc/systemd/system/varnish.service.d/override.conf
+  path: /var/tmp/varnish-override.conf
```

Comment:
no reason to change this, it can be installed before the package without issue

Reply:
The `/etc/systemd/system/varnish.service.d` folder doesn't exist at this point. I ran this and experienced issues.

How would you tackle this?

Reply:
[docs](https://cloudinit.readthedocs.io/en/latest/reference/modules.html#write-files) say:
> Parent folders in the path are created if absent. 

so it's not an issue?

Reply:
You're right @gquintard, it wasn't an issue.